crisischeckin/crisicheckinweb/Controllers/VolunteerController.cs

```diff
@@ -53,7 +53,7 @@ public ActionResult SendMessageToVolunteersByDisaster(SendMessageToAllVolunteers
                 return View("CreateMessage", model);
             }
             var sender = model.DisasterName + " - Coordinator";
-            var recipientCriterion = new RecipientCriterion(model.DisasterId, model.ClusterId, model.ClusterCoordinatorsOnly, model.CheckedInOnly);
+            var recipientCriterion = new RecipientCriterion(model.DisasterId, model.SelectedClusterIds, model.ClusterCoordinatorsOnly, model.CheckedInOnly);
             var message = new Message(model.Subject, model.Message);
             _messageSvc.SendMessageToDisasterVolunteers(message, recipientCriterion, sender);

```

Comment:
We need to look at this interface and its implementation to make sure it manages the updated recipient Criterion.